client/reader/list-stream/header.jsx

```diff
@@ -38,7 +39,20 @@ const ListStreamHeader = ( {
 			</span>
 
 			<div className="list-stream__header-details">
-				<h1 className="list-stream__header-title">{ title }</h1>
+				<div className="list-stream__header-title">
+					<h1>{ title }</h1>
+					{ ! isPublic && (
+						<div
+							className="list-stream__header-title-privacy"
+							title={ translate( 'Private list' ) }
+						>
+							<Gridicon icon="lock" size={ 18 } />
+							<span className="list-stream__header-title-privacy-label screen-reader-text">
+								{ translate( '(private)' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **37** times:
`translate( 'Private', { context: 'Editor: Radio label to set post to private'} )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader-list-indicate-private-list)